### PR TITLE
Only load new relic when there is a license key

### PR DESCRIPTION
**Pull request by orta**
### Before
screen shot 2017-09-18 at 10 16 48 am

### After

screen shot 2017-09-18 at 10 21 14 am

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import Bluebird from "bluebird"
 import newrelic from "artsy-newrelic"
+import Bluebird from "bluebird"
```
Comment (orta) [inline index.js]: One of the newrelic logs is specifically that newrelic needs to load before other modules, I don't know if this is a case of "we have to do this" - so I'm highlighting this

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
Fixes #744

**Comment by craigspaeth** [comment]
👍  LGTM. FWIW I wrote a [tiny wrapper](https://github.com/artsy/artsy-newrelic) that does this among other conventions around how we use NR (report uncaught exceptions, timeout & report). Feel free to merge when 💚

**Comment by orta** [comment]
Looks like that already supports the suppression, I was only seeing it because my ENV had the key - closing, no point in adding duplicated checks 👍

**Pull request closed without merging**